app/navigation/navigation-activity/src/main/kotlin/com/hedvig/android/navigation/activity/ActivityNavigator.kt

```diff
@@ -10,14 +10,26 @@ import com.hedvig.android.core.common.android.tryOpenPlayStore
 import com.hedvig.android.logger.LogPriority
 import com.hedvig.android.logger.logcat
 
-class ActivityNavigator(
+interface ActivityNavigator {
+  fun navigateToMarketingActivity()
+
+  fun openAppSettings(context: Context)
+
+  fun navigateToLoggedInScreen(context: Context, clearBackstack: Boolean = true)
+
+  fun tryOpenPlayStore(context: Context)
+
+  fun openWebsite(context: Context, uri: Uri)
+}
+
+class ActivityNavigatorImpl(
```

Comment:
Would it be worth it, now that it's an impl make it internal, make a separate di module in navigation and expose out only the interface?